ex03.ex

```diff
@@ -59,7 +59,15 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+#   Edge case for empty list
```

Comment:
# +2

I love the fact that you're explicitly thinking about the termination conditions.